Remove link pointing to Keith Peters' Twitter

meetanoopverma opened the pull request:
<!-- 🙌 Thanks for contributing to MDN Web Docs. Adding details below will help us to merge your PR faster. -->
This PR is intended to replace the outdated link pointing to the inactive profile of the Keith Peters with the new updated link.

<!-- ✍️ Summarize your changes in one or two sentences -->

To redirect every reader to the location where they can find more relevant resources to learn.

<!-- ❓ Why are you making these changes and how do they help readers? -->

It redirects them to get connected with the person concerned and engage in a learning pursuit.

<!-- 🔗 Link to release notes, vendor docs, bug trackers, source control, or other places providing more context -->

### Related issues and pull requests

"Fixes #28336"
"Relates to #28336"

meetanoopverma committed:
This commit is to update the hyperlink pointing to Keith Peters

The earlier link points to the inactive social profile of the Keith Peters. Now the updated one points to the active one.

meetanoopverma commented:
Kindly review and merge this PR.

wbamberg reviewed (dismissed):
Thanks for your PR! The suggestion from @hamishwillee in https://github.com/mdn/content/issues/28336#issuecomment-1664865407 was to remove the name and link.

meetanoopverma committed:
Name Keith Peters and the associated link has been removed

As per the suggestion, The name and the associated link for the twitter handle have been removed.

meetanoopverma reviewed (commented):
As per the suggestion, changes have been effected.

meetanoopverma committed:
Merge branch 'main' into meetanoopverma-patch-1-1

bsmth commented on files/en-us/learn/javascript/index.md:
```diff
@@ -68,4 +68,4 @@ This topic contains the following modules, in a suggested order for working thro
   - : An excellent resource for aspiring web developers — Learn JavaScript in an interactive environment, with short lessons and interactive tests, guided by automated assessment. The first 40 lessons are free.
 - [Coding math](https://www.youtube.com/user/codingmath)
-  - : An excellent series of video tutorials to teach the math you need to understand to be an effective programmer, by [Keith Peters](https://twitter.com/bit101).
+  - : An excellent series of video tutorials to teach the math you need to understand to be an effective programmer.
```
```suggestion
  - : An excellent series of video tutorials to teach the math you need to understand to be an effective programmer, by [Keith Peters](https://www.bit-101.com/blog/about-me/).
```

bsmth committed:
Update files/en-us/learn/javascript/index.md

bsmth commented:
Thanks @meetanoopverma - I've added a link to https://www.bit-101.com/blog/about-me/ which contains links to socials / other resources.
